crates/sui-indexer/tests/integration_tests.rs

```diff
@@ -2,27 +2,34 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // integration test with standalone postgresql database
-#[cfg(feature = "pg_integration")]
+// #[cfg(feature = "pg_integration")]
```

Comment:
question: why comment instead of just deleting?

Reply:
oops